config/settings.py

```diff
@@ -58,6 +58,7 @@
     "django.middleware.security.SecurityMiddleware",
     "whitenoise.middleware.WhiteNoiseMiddleware",
     "django.contrib.sessions.middleware.SessionMiddleware",
+    "movie_search.middleware.TMDBApiMiddleware",
```

Comment:
I checked this with gpt and it's safer to add it at least after the common and csrf middlewares

Reply:
> Here, TMDBApiMiddleware is positioned after the AuthenticationMiddleware and CsrfViewMiddleware, ensuring that it operates in a fully authenticated and secure context, which is generally a safer and more robust configuration.